zsh/functions/tat

```diff
@@ -30,8 +30,9 @@ create_if_needed_and_attach() {
 }
 
 tat() {
-  path_name="$(basename "$PWD" | tr . -)"
-  session_name=${1-$path_name}
+  parent="$(basename $(dirname "$PWD"))"
```

Comment:
They're not always clients, so I think I'll stick with `parent`. Updated for quotes in dccd86a57ad7a1a5533c8afe240c91f5fc575ef3